redbot/pytest/downloader.py

```diff
@@ -163,14 +162,7 @@ def _init_test_repo(destination: Path):
 async def _session_git_repo(tmp_path_factory, event_loop):
     # we will import repo only once once per session and duplicate the repo folder
     repo_path = tmp_path_factory.mktemp("session_git_repo")
-    repo = Repo(
-        name="redbot-testrepo",
-        url="",
-        branch="master",
-        commit="",
-        folder_path=repo_path,
-        loop=event_loop,
-    )
+    repo = Repo(name="redbot-testrepo", url="", branch="master", commit="", folder_path=repo_path,)
```

Comment:
I hate that black now doesn't remove the trailing comma automatically >.>
```suggestion
    repo = Repo(name="redbot-testrepo", url="", branch="master", commit="", folder_path=repo_path)
```